core/sql/optimizer/UdfDllInteraction.cpp

```diff
@@ -1214,6 +1214,29 @@ NABoolean TMUDFInternalSetup::setTypeInfoFromNAType(
           switch (length)
             {
               // SMALLINT, INT, LARGEINT, NUMERIC, signed and unsigned
+            case 1:
+              if (CmpCommon::getDefault(TRAF_TINYINT_SPJ_SUPPORT) == DF_ON)
```

Comment:
Some support still need to be added to SPJ/T2 driver path for tinyint.
If this cqd is turned on, it continues to the path of processing tinyint.
Since tinyint support is not yet in for spj/t2, an error is currently returned. 
This is not an externalized cqd.